doc/source/ray-contribute/getting-involved.rst

```diff
@@ -246,13 +246,67 @@ check recent tests known to be flaky.
 
 .. _`CI`: https://github.com/ray-project/ray/tree/master/ci
 
+.. _api-stability:
+
 API stability
 -------------
 
-Ray provides stability guarantees for its public APIs in Ray core and libraries. The level of stability provided depends on how the API is annotated.
+Ray provides stability guarantees for its public APIs in Ray core and libraries, which are decorated/labeled accordingly.
+
+An API can be labeled:
+
+* :ref:`PublicAPI <public-api-def>`, which means the API is exposed to end users. PublicAPI has three sub-levels (alpha, beta, stable), as described below.
+* :ref:`DeveloperAPI <developer-api-def>`, which means the API is explicitly exposed to *advanced* Ray users and library developers
+* :ref:`Deprecated <deprecated-api-def>`, which may be removed in future releases of Ray.
+
+Ray's PublicAPI stability definitions are based off the `Google stability level guidelines <https://google.aip.dev/181>`_, with minor differences:
```

Comment:
Should we also unify with RLlib stability annotations? (delete that page / direct here)

Reply:
I've created an issue for Sven/team to migrate their annotations #25628 .